### ISSUE TITLE
Denial of Service Node Fetch

### ISSUE THREAD

**Issue by erwanriou** [ryandrewjohnson/react-localize-redux#209]
Hello,

Just to inform that we currently have a small issue in one of the dependencies. 
https://npmjs.com/advisories/1556

**Comment by s100** [comment]
That advisory comes from `node-fetch`, which is used by `isomorphic-fetch`, which is used by `fbjs`, which is used by `create-react-context@0.2.3`, which is used by `react-localize-redux`. `create-react-context` [stopped using `fbjs` entirely](https://github.com/jamiebuilds/create-react-context/pull/22) in `create-react-context@0.3.0`, which was released several years ago. So, all `react-localize-redux` needs to do is upgrade to `create-react-context@0.3.0`.

[There is a PR open to do this](https://github.com/ryandrewjohnson/react-localize-redux/pull/212) but despite being essentially a one-line change it has been open since November 2020 with no activity. And there has not been a new release of `react-localize-redux` for more than two years.

In other words it doesn't look like this is going to be fixed. Our options are to either manually ignore the advisory or stop using `react-localize-redux`.
